code/_globalvars/lists/rcd.dm

```diff
@@ -5,6 +5,7 @@ GLOBAL_LIST_INIT(rcd_designs, list(
 		//Walls & Windows
 		"Structures" = list(
 			list(RCD_DESIGN_MODE = RCD_TURF, RCD_DESIGN_PATH = /turf/open/floor/plating/rcd),
+			list(RCD_DESIGN_MODE = RCD_DECONSTRUCT, RCD_DESIGN_PATH = /turf/open/floor/plating/deconstruct),
```

Comment:
Absolutely not. This list is meant for solid structures that are to be created, not abstract actions like deconstruct. Its snowflacky and doesn't  belong here

Just add the deconstruct action to the mech ui as an toggle button. The primary rcd ui should not be affected just because the mech wants this